keys.js

```diff
@@ -0,0 +1,11 @@
+// console.log('this is loaded');
+
+var twitterKeys = {
+    consumer_key: 'F7CW1UpnYde6YWMjvtrXxHwlv',
+    consumer_secret: 'pENB1Ezpr1tixiI230p4pseTz1H8HCcRZTgnclMm5LGHlwsOa9',
+    access_token_key: '49535583-uOasGmrMr5Wn19FC0LXTztS1ymYUODwRZjq0ZSswO',
+    access_token_secret: '8byY6zICD1CRf7g1pQbxyW8AD1tEqViZkjxmVwlp1sslw'
+}
+
+module.exports = twitterKeys;
+
```

Comment:
I would recommend creating your module instance here and then exporting that into your liri file.